docs/components/board/pca9685.md

```diff
@@ -79,14 +75,5 @@ The following attributes are available for `pca9685` boards:
 <!-- prettier-ignore -->
 | Name | Type | Inclusion | Description |
 | ---- | ---- | --------- | ----------- |
-| `board_name` | string | Optional | The `name` of the board with GPIO pins your `pca9685` is [connected to](https://learn.adafruit.com/16-channel-pwm-servo-driver/hooking-it-up). |
-| `i2c_name` | string | Optional | The `name` of the `i2c` your `pca9685` is connected to, as configured in your main board's section of the config file. |
-| `i2c_bus` | string | Optional (recommended) | The index of the board with GPIO pins your `pca9685` is connected to. Often a number. <br> Example: `”1”` |
+| `i2c_bus` | string | **Required** | The index of the I<sup>2</sup>C bus on the board with GPIO pins your `pca9685` is connected to. Often a number. <br> Example: `”1”` |
```

Comment:
:+1: I like the way you've phrased this, making clear that it's the I2C bus that's important, instead of the index of the board, whatever that meant. Nicely done!